components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.h

```diff
@@ -24,6 +24,20 @@
  Delegate for MDCBottomDrawerContainerViewController.
  */
 @protocol MDCBottomDrawerContainerViewControllerDelegate <NSObject>
+
+/**
+This method is called when the MDCBottomDrawerContainerViewControllerDelegate needs to update the
+appearance of the scrim.
```

Comment:
can you please elaborate here as in what cases there is a need to update the appearance of the scrim